core/iwasm/common/wasm_runtime_common.c

```diff
@@ -1400,6 +1406,7 @@ wasm_runtime_load_from_sections(WASMSection *section_list, bool is_aot,
             LOG_DEBUG("WASM module load failed from sections");
```

Comment:
Seems we had better also set `args.is_binary_freeable = false` in wasm_runtime_load, see L1385, though `LoadArgs args = { 0 };` already set it to false?